pkg/test/framework/components/istio/operator.go

```diff
@@ -283,21 +272,39 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		if errs := errG.Wait(); errs != nil {
 			return nil, fmt.Errorf("%d errors occurred deploying remote clusters: %v", errs.Len(), errs.ErrorOrNil())
 		}
-	}
 
-	if env.IsMultinetwork() {
-		// enable cross network traffic
-		for _, cluster := range env.KubeClusters {
-			if err := i.applyCrossNetworkGateway(cluster); err != nil {
+		if env.IsMulticluster() {
+			// For multicluster, configure direct access so each control plane can get endpoints from all
+			// API servers.
+			if err := i.configureDirectAPIServerAccess(ctx, env, cfg); err != nil {
 				return nil, err
 			}
+			for _, cluster := range env.KubeClusters {
+				if !(env.IsControlPlaneCluster(cluster) || env.IsConfigCluster(cluster)) {
+					// remote clusters only need this gateway for multi-network purposes
+					if i.ctx.Environment().IsMultinetwork() {
+						if err := i.deployEastWestGateway(cluster); err != nil {
+							return i, err
+						}
+					}
+				}
+			}
+		}
+		if env.IsMultinetwork() {
+			// enable cross network traffic
+			for _, cluster := range env.KubeClusters {
+				if err := i.applyCrossNetworkGateway(cluster); err != nil {
```

Comment:
Let's rename this `i.exposeUserServices(cluster)`. The name is not immediately clear.

Reply:
yes, agree this will align our multi-cluster doc as well.